src/lib/tls/msg_client_hello.cpp

```diff
@@ -513,12 +531,22 @@ Client_Hello_12::Client_Hello_12(Handshake_IO& io,
    if(policy.support_cert_status_message())
       m_data->extensions().add(new Certificate_Status_Request({}, {}));
 
-   auto supported_groups = std::make_unique<Supported_Groups>(policy.key_exchange_groups());
-   if(!supported_groups->ec_groups().empty())
+   // RFC 8446 4.2.3
```

Comment:
Same as above: Not in RFC 8446 but in [RFC 8422 4](https://www.rfc-editor.org/rfc/rfc8422#section-4).